src/hooks/useAppContext.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+
+/**
+ * @typedef {import("../appContext").AppContextObject} AppContextObject
+ */
+
+/**
+ * Handle the state management for the modal path data that is shared via context.
+ *
+ * @returns {AppContextObject} Modal path state object.
```

Comment:
The jsdoc description needs to be updated

Reply:
Done.